test/configCases/rule-set/relative-path/webpack.config.js

```diff
@@ -0,0 +1,28 @@
+/** @type {import("../../../../").Configuration} */
+module.exports = {
+	module: {
+		rules: [
+			{
+				test: /\.js$/,
+				include: (resource, { descriptionData, relativePath }) =>
+					descriptionData &&
+					descriptionData.name === "fake-package" &&
+					descriptionData.version === "1.0.0" &&
+					relativePath === "./lib/button.js",
+				loader: "./loader",
+				options: {
+					value: "matched-by-relative-path"
+				}
+			},
+			{
+				test: /\.js$/,
+				include: (resource, { descriptionData, relativePath }) =>
+					descriptionData && descriptionData.name === "fake-package",
```

Comment:
Let's add a test case for `test` and `exclude`, thank you

Reply:
> Let's add a test case for `test` and `exclude`, thank you

OK, I added